tests/test-pg-error-view.json

```diff
@@ -0,0 +1,61 @@
+{
+    "webserver": {
+        "views/test-pg-error": {
+            "view": "fostgres.control.pg-error",
+            "configuration": {
+                "execute": {
+                    "view": "fostgres.sql",
+                    "configuration": {
+                        "dbname": [
+                            "request",
+                            "headers",
+                            "__pgdsn",
+                            "dbname"
+                        ],
+                        "sql": [
+                            {
+                                "return": "object",
+                                "path": [1],
+                                "GET": "SELECT * FROM users WHERE username=$1",
+                                "PUT": [
+                                    {
+                                        "table": "users",
+                                        "columns": {
+                                            "username": {
+                                                "key": true,
+                                                "source": 1
+                                            },
+                                            "email": {}
+                                        }
+                                    }
+                                ]
+                            },
+                            {
+                                "return": "object",
+                                "path": [1, "/friends", 2],
+                                "GET": "SELECT * FROM friends WHERE username=$1 and friend=$2",
+                                "PUT": [
+                                    {
+                                        "table": "friends",
+                                        "columns": {
+                                            "username": {
+                                                "key": true,
+                                                "source": 1
+                                            },
+                                            "friend": {
+                                                "key": true,
+                                                "source": 2
+                                            },
+                                            "friend_type": {}
+                                        }
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                },
+                "": "fost.response.501"
```

Comment:
I would use a different error code here because the `not_implemented` exception will also give a 501 even in tests.